ultraplot/axes/plot.py

```diff
@@ -4363,7 +4405,13 @@ def _apply_fill(
         **kwargs,
     ):
         """
-        Apply area shading.
+        Apply area shading (fill_between / fill_betweenx) with seaborn helper tagging.
+
+        We tag seaborn-generated confidence interval / helper polygons as synthetic
+        unless the user explicitly supplies a label (label/labels/value/values or
+        shadelabel/fadelabel passed through error shading). Synthetic artists are
+        marked with _ultraplot_synthetic=True and given an underscore label so they
+        are ignored by legend collection.
```

Comment:
This doc string is very helpful! I think "seaborn" -> "externally-created" since now the mechanism is for any external package?

Reply:
It will work for others as well (but I am not fully aware of the ins and out of other packages if they internally tag some plots. We can make it more general and then add seaborn as a specific example.

Reply:
I cleaned up the docstring.